lib/elements/fill.py

```diff
@@ -28,9 +29,25 @@ class InvalidShapeError(ValidationError):
     name = _("Border crosses itself")
     description = _("Fill: Shape is not valid.  This can happen if the border crosses over itself.")
     steps_to_solve = [
-        _('* Path > Union (Ctrl++)'),
-        _('* Path > Break apart (Shift+Ctrl+K)'),
-        _('* (Optional) Recombine shapes with holes (Ctrl+K).')
+        _("1. Inkscape has a limit to how far it lets you zoom in.  Sometimes there can be a little loop, "
+          "that's so small, you can't see it, but Ink/Stitch can. It's especially common for Inkscape's "
+          "Trace Bitmap to produce those tiny loops."),
+        _("* Delete the node"),
+        _("* Or try to adjust it's handles"),
+        _("2. If you can actually see a loop, run the following commands to seperate the crossing shapes:"),
+        _("* Path > Union (Ctrl++)"),
+        _("* Path > Break apart (Shift+Ctrl+K)"),
+        _("* (Optional) Recombine shapes with holes (Ctrl+K).")
+    ]
+
+
+class InvalidSVGObjectError(ValidationError):
+    name = _("Object conversion failed")
+    description = _("Fill: Automated path-creation for this object failed.  Convert it to a path (Shift+Ctrl+C).")
```

Comment:
Out of curiosity, when does this happen?  Have you seen it happen?

Reply:
Actually, yes I have.
Circles with fill stitching like to cause issues like this - even when you convert them to a path. I got the feeling, that our conversion sometimes works slightly better than the original one. Usually it can be solved by changing the size just a little tiny bit or move the object just a little. Could this behavior be improved in the fill stitch algorithm?

Reply:
I've definitely had problems after converting a circle to a path -- in Inkscape.  But does your new automatic conversion code have the same problem?

> Usually it can be solved by changing the size just a little tiny bit or move the object just a little. Could this behavior be improved in the fill stitch algorithm?

I'm gonna give this some thought.  It would be pretty nice if we could ignore tiny, invisible loops...

Reply:
The conversion from circles to paths has almost the same issue as Inkscape's conversion has...
Oh yes, it would be so nice, if we could just ignore those tiny loops!
But it doesn't seem to be an easy task?!